### DISCUSSION TITLE
反馈

### DISCUSSION THREAD

**Discussion by Anhai0501** [gkd-kit/gkd#140] [Ideas]
自定义添加规则有点困难 纯小白 不知道咋弄（个别应用的开屏广告适配不全 不能跳过 开李跳跳可以） 
然后就是感觉内存占用有点高

**Comment by lisonge** [comment]
> 自定义添加规则有点困难 纯小白 不知道咋弄

https://github.com/gkd-kit/subscription/blob/main/CONTRIBUTING.md

---

> 感觉内存占用有点高

后期会优化
